src/data/index.ts

```diff
@@ -218,13 +218,13 @@ export const exportToBackend = async (
       // of queryParam in order to never send it to the server
       url.hash = `json=${json.id},${exportedKey.k!}`;
       const urlString = url.toString();
-
       window.prompt(`🔒${t("alerts.uploadedSecurly")}`, urlString);
+    } else if (json.error_class === "RequestTooLargeError") {
+      window.alert(t("alerts.couldNotCreateShareableLinkTooBig"));
     } else {
       window.alert(t("alerts.couldNotCreateShareableLink"));
     }
   } catch (error) {
```

Comment:
```suggestion
  } catch (error) {
  	console.error(error);
```

We want to keep this so we log unexpected errors to our Sentry

Reply:
Husky doesn't accept console statement
![2020-10-18_10-08](https://user-images.githubusercontent.com/65488471/96361917-f1068600-1129-11eb-83a1-36c539577df3.png)